src/main/java/org/ow2/proactive/catalog/util/LinkUtil.java

```diff
@@ -85,12 +86,12 @@ public static Link createLink(String bucketName, String name, LocalDateTime comm
     public static Link createLink(String bucketName, String name)
             throws NotAuthenticatedException, AccessDeniedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)